### PR TITLE
Query

### LINKED ISSUE #54: Implement a method to do queries on pay services
In a some moment, the magento will need to execute queries on pay like search a payment by hash to update the order status.

### PR DESCRIPTION
**Pull request by gpressutto5**
Closes #54

### CODE REVIEW

File: src/Services/Adapters/PaymentInfoAdapter.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+     *
+     * @param string $type
+     * @param string $code
+     * @param Config $config
+     */
+    public function __construct($type, $code, Config $config)
+    {
+        $this->type = $type;
+        $this->code = $code;
+        parent::__construct($config);
```
Comment (cezarlz) [inline src/Services/Adapters/PaymentInfoAdapter.php]: Please, implement the facade here

File: src/Services/PaymentInfo.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        $this->config = $config;
+        $this->client = $this->client ?: new Client();
+    }
+
+    /**
+     * @param string $hash
+     * @param boolean   $isSandbox
+     * @return array
+     */
+    public function getPaymentInfoByHash($hash, $isSandbox = null)
```
Comment (cezarlz) [inline src/Services/PaymentInfo.php]: Why getPaymentInfoByHash if the class is PaymentInfo. Maybe `getByHash` is good.

File: src/Services/PaymentInfo.php [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        return $this->getResponse('hash', $hash, $isSandbox);
+    }
+
+    /**
+     * @param string $merchantPaymentCode
+     * @param boolean   $isSandbox
+     * @return array
+     */
+    public function getPaymentInfoByMerchantPaymentCode($merchantPaymentCode, $isSandbox = null)
+    {
```
Comment (cezarlz) [inline src/Services/PaymentInfo.php]: Same here.

File: src/Services/PaymentInfo.php (hunk 2 above)
Comment (gpressutto5) [inline src/Services/PaymentInfo.php]: `findByHash` is better

File: src/Services/PaymentInfo.php (hunk 3 above)
Comment (gpressutto5) [inline src/Services/PaymentInfo.php]: Same here

**Comment by cezarlz** [comment]
@gpressutto5 Could you update the Wiki adding examples of how to use it? Tks!

**Pull request merged**